src/app/components/edit-user-card/edit-user-card.component.ts

```diff
@@ -105,6 +113,17 @@ export class EditUserCardComponent implements OnInit {
     }
   }
 
+  onImageMissing(image: HTMLImageElement): void {
+    image.src = this.fallbackImageUrl;
+  }
+
+  isCardInvalid(): boolean {
```

Comment:
This could be a variable instead of a getter (is run everytime on change detection)
```isCardInvalid = !(
       this.userForm.valid &&
       !this.image?.nativeElement.src.endsWith(this.fallbackImageUrl)
     );```